runtime-modules/proposals/codex/src/tests/mod.rs

```diff
@@ -621,25 +621,15 @@ fn create_set_validator_count_proposal_common_checks_succeed() {
 #[test]
 fn create_set_validator_count_proposal_failed_with_invalid_validator_count() {
     initial_test_ext().execute_with(|| {
-        assert_eq!(
```

Comment:
Could you briefly explain what was broken about this test that required these changes?

Reply:
I noticed test duplication, which is likely to be the merge artifact. During the test debugging I changed the stake to the required value for this proposal type (it was unnecessary). And finally, the test was broken because of the default `staking::MinValidatorCount` changes. It looks like it is not set by default when using the latest Substrate libraries.